### ISSUE TITLE
composer install failed

### ISSUE THREAD

**Issue by alexschwarz89** [vinkla/instagram#31]
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for vinkla/instagram ^4.1 -> satisfiable by vinkla/instagram[4.1.0].
    - Conclusion: remove graham-campbell/manager v2.2.0
    - Conclusion: don't install graham-campbell/manager v2.2.0
    - vinkla/instagram 4.1.0 requires graham-campbell/manager ^2.3 -> satisfiable by graham-campbell/manager[v2.3.0, v2.3.1, v2.4.0].
    - Can only install one of: graham-campbell/manager[v2.3.0, v2.2.0].
    - Can only install one of: graham-campbell/manager[v2.3.1, v2.2.0].
    - Can only install one of: graham-campbell/manager[v2.4.0, v2.2.0].
    - Installation request for graham-campbell/manager (locked at v2.2.0) -> satisfiable by graham-campbell/manager[v2.2.0].
```

**Comment by vinkla** [comment]
Please provide more information. What's versions are you using and what other packages? Just pasting an error won't get you far.

**Comment by alexschwarz89** [comment]
Solved it when specifiyng ^v2.3.1 of graham-campbell/manager.

These are the other packages in composer.json

` "require": {
        "php": ">=5.5.9",
        "laravel/framework": "5.1.*",
        "barryvdh/laravel-ide-helper": "^2.1",
        "doctrine/dbal": "^2.5",
        "rydurham/sentinel": "^2.2",
        "detectlanguage/detectlanguage": "^2.0",
        "facebook/php-sdk-v4": "4.0.*",
        "laravelcollective/html": "^5.1",
    },
    "require-dev": {
        "fzaninotto/faker": "~1.4",
        "mockery/mockery": "0.9.*",
        "phpunit/phpunit": "~4.0",
        "phpspec/phpspec": "~2.1"
    },`

**Comment by vinkla** [comment]
Okay, good. I'll close this then.
